dap_aria_mapping/utils/semantics.py

```diff
@@ -0,0 +1,101 @@
+import numpy as np
```

Comment:
These are really useful - would name the script entity_selection.py as I could see these utils being used for a variety of use cases, not just the semantic models

Reply:
Sure! These are originally in the semantics utils in #9, hence me trying to keep them consistent across branches. But will just make sure I move them to entity_selection.